### PR TITLE
feat(avo-1912): add empty pupil collection state, fix getResponses query

### PR DESCRIPTION
**Pull request by bertyhell**
https://meemoo.atlassian.net/browse/AVO-1912

image

### CODE REVIEW

File: src/assignment/views/AssignmentResponseEdit/tabs/AssignmentResponsePupilCollectionTab.tsx [hunk 1]
```diff
@@ -182,6 +190,28 @@ const AssignmentResponsePupilCollectionTab: FunctionComponent<
...
+								type="inline-link"
+								label={t('assignment/views/assignment-response-edit/tabs/assignment-response-pupil-collection-tab___zoeken')}
+								onClick={() =>
+									setTab(ASSIGNMENT_RESPONSE_CREATE_UPDATE_TABS.SEARCH)
+								}
+							/>{' '}
+							{t('assignment/views/assignment-response-edit/tabs/assignment-response-pupil-collection-tab___om-fragmenten-toe-te-voegen-of-druk-op-de-plus-knop-hierboven-als-je-tekstblokken-wil-aanmaken')}
+						</p>
+					</Flex>
+				</Container>
```
Comment (ian-emsens-shd) [inline src/assignment/views/AssignmentResponseEdit/tabs/AssignmentResponsePupilCollectionTab.tsx:214]: Split to separate component

**Review by ian-emsens-shd — CHANGES_REQUESTED** [review]

File: src/shared/components/StickySaveBar/StickySaveBar.scss [hunk 2]
```diff
@@ -5,6 +5,7 @@
 	flex-direction: column;
 	flex-grow: 1;
 	justify-content: space-between; // Keep the save indicator at the bottom
+	min-height: calc(100vh - 40rem); // Ensure save bar stays at the bottom when not enough content
```
Comment (ian-emsens-shd) [inline src/shared/components/StickySaveBar/StickySaveBar.scss:8]: These lines do the same thing; are all the parent containers around `.c-container--sticky-save-bar-wrapper` filling their own parents so `space-between` can do its thing?

Comment (bertyhell) [reply src/shared/components/StickySaveBar/StickySaveBar.scss:8]: The min-height seems to be needed to make the container large enough when not enough content is on the page

This is what i think is happening:
image

File: src/shared/components/EmptyStateMessage/EmptyStateMessage.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+import React, { FC, ReactNode } from 'react';
+
+interface EmptyStateMessageProps {
+	imgSrc: string;
+	imgAlt: string;
+	title: string;
+	message: ReactNode;
+}
+
+const EmptyStateMessage: FC<EmptyStateMessageProps> = ({ imgSrc, imgAlt, title, message }) => {
```
[the remark this answers is not in the document]
Comment (bertyhell) [reply src/shared/components/EmptyStateMessage/EmptyStateMessage.tsx]: moved imgSrc and imgAlt to defaults, the rest is to specific to move to a default

**Pull request merged**